### ISSUE TITLE
Inbox rules magically disappear

### ISSUE THREAD

**Issue by bedhub** [tutao/tutanota#1073]
**Bug in web app**
Users have reported that inbox rules that they have created are not present at the next day. This can happen because of missing syn between different clients.

**To Reproduce**
1. Log in with the same user account in two different clients e.g. Firfox and Chrome browser
1. Disable network access in Chrome browser so the user is offline
2. Create new inbox rule in Firefox
3. Enable network access in Chrome and create another inbox rule 
-> here the first inbox rule is not visible. When creating the new inbox rule the first one will be removed because all rules are stored in the same instance. The client does not  synchronize this particular instance. Other instances like mails will receive an update event after reconnect.

**Comment by geocfu** [comment]
I can confirm this issue.

Yesterday i created 8 inbox rules and today I had 3. This is the second time that it happens. Also, note that i had this behavior on inbox rules that point to custom folders. 

The weird thing is that right after I created my custom inbox rules on desktop, as soon as i logged in my iPhone, the rules where there, working as expected. But the next day, only 3 inbox rules appear on all devices.

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Browser [Brave]
 - Version [3.46.2]

**Smartphone (please complete the following information):**
 - Device: [iPhone 5s]
 - OS: [iOS 12.1.3]
 - Browser [Native Tuta App]
 - Version [3.46.3]

**Comment by mpfau** [comment]
Occured again for a customer

**Comment by bedhub** [comment]
Some more information about this bug observed by a user:

"As I indicated before, the "steps to reproduce" ...  don't apply to my situation, and the Inbox Rules continue to disappear. I am only using one browser (Chromium based), with no other instances open anywhere else. I leave my browser open pretty much all the time. Next day I check, and all the rules I created are gone."

Inbox rules are currently stored on the TutanotaProperties instance together with other properties. We need to check all places were we write TutanotaProperties and ensure that there we don't write an outdated version of that instance.
